packages/principal/src/index.test.ts

```diff
@@ -24,4 +24,21 @@ describe('Principal', () => {
   it('errors out on parsing invalid characters', () => {
     expect(() => Principal.fromText('Hello world!')).toThrow();
   });
+
+  it('compares principals correctly', () => {
+    const anonymous = Principal.anonymous();
+    const principal1 = Principal.fromText('ryjl3-tyaaa-aaaaa-aaaba-cai');
+    const principal2 = Principal.fromText('ivg37-qiaaa-aaaab-aaaga-cai');
+    for (let p of [anonymous, principal1, principal2]) {
```

Comment:
lint error here: the tests are failing because this is a `let` instead of `const`.